### DISCUSSION TITLE
Extended installation documentation

### DISCUSSION THREAD

**Discussion by Bckstage** [logto-io/logto#1779] [Q&A] [answered]
Hi there!

I really do like LogTo and I am very excited for testing it. Would you please go into further details of setting up LogTo? If you are running a docker (or non-docker) instance outside of a "localhost", things get complicated very quickly.

The issues I have encountered (and yes, I am somewhat a noob..):

**Docker**
Whilst setting up a docker images through docker-compose on a Ubuntu Docker server (in this case on my own physical server hosted in a datacenter), I keep banging my head around the SSL-issues. You have described the issue but the solution is very cryptic.

**Download script**
It's pretty straight forward, until the point Node throws a giant 'Nope' with a Permission Denied for 'node-pre-gyp'.

The installation script automatically changes the permission to the user '1001' with the group '121' to the main folder 'logto', which causes the Permission Denied (I think?).

Are these things I can alter from my side or does this also require a change from your end with future deployments? I would really love it if you edit the installation documentation with more "noob"-friendly instructions.

**Comment by gao-sun** [comment]
sorry we missed the discussion part since it has no notification to us.. will take a look soon.

edit: we just subscribed GitHub discussion in our working Slack channel.

**Reply by gao-sun** [reply]
btw, we're also glad you like Logto :-)

**Comment by charIeszhao** [comment]
Hey @komcommy, the documentation is updated and the docker command sample and a nginx reverse proxy sample are provided right now.

[Get Started](https://docs.logto.io/docs/tutorials/get-started/) -> Docker:
<img width="989" alt="image" src="https://user-images.githubusercontent.com/12833674/188277181-edf1c57c-7401-4554-8eb8-a5857d6189b9.png">

Nginx reverse proxy:
https://docs.logto.io/docs/recipes/deployment/#reverse-proxy

Please check them out and let us know if there's still anything we can improve.

Thanks!

**Comment by charIeszhao** [comment, marked as answer]
Hey @komcommy, the documentation is updated and the docker command sample and a nginx reverse proxy sample are provided right now.

[Get Started](https://docs.logto.io/docs/tutorials/get-started/) -> Docker:
<img width="600" alt="image" src="https://user-images.githubusercontent.com/12833674/188277181-edf1c57c-7401-4554-8eb8-a5857d6189b9.png">

Nginx reverse proxy:
https://docs.logto.io/docs/recipes/deployment/#reverse-proxy

Please check them out and let us know if there's still anything we can improve.

Btw, the one-liner issue is acknowledged on Windows platform. Are you using Windows? And we have already asked the community for help to develop a Windows compatible one-liner installation script #1607. Stay tuned!

Thanks!
